lib/landrush/cap/linux/configured_dns_servers.rb

```diff
@@ -4,9 +4,9 @@ module Linux
       module ConfiguredDnsServers
         def self.configured_dns_servers(machine)
           return @dns_servers if @dns_servers
-          machine.communicate.sudo('cat /etc/resolv.conf | grep ^nameserver') do |type, data|
+          machine.communicate.sudo('sed -ne \'s/^nameserver \([0-9.]*\)$/\1/p\' /etc/resolv.conf') do |type, data|
```

Comment:
So why this change. What is the actual issue you are addressing? The _cat_ command did return something wrong on CentOS? I